packages/datagovmy-ui/src/components/Combobox/option.tsx

```diff
@@ -73,6 +73,12 @@ function ComboOptionInner<T>(
                   width={28}
                   height={18}
                   alt={option.value as string}
+                  style={{
+                    width: "auto",
+                    maxWidth: "28px",
+                    height: "auto",
+                    maxHeight: "28px",
+                  }}
                 />
```

Comment:
why are we hardcoding the styles here?

Reply:
for odd-shaped flags, like Switzerland or party flags like SETIA

Reply:
Ok, actually kan there's a better way to do this. Why dont you turn this `ImageWithFallback` part into a prop function: `(value) => ReactNode`. Value refers to `option.value`. Make it flexible. If we use images other than flag, this component is useless

So you can remove `imageSource`